src/btree/bt_split.c

```diff
@@ -30,6 +30,8 @@ typedef enum {
 	WT_ERR_RETURN				/* Clean up and return error */
 } WT_SPLIT_ERROR_PHASE;
 
+static void __page_split_timing_stress(WT_SESSION_IMPL *);
```

Comment:
The function definition can be moved before the first function call, and you won't need a separate declaration.